app/services/api/webflow-helpers.ts

```diff
@@ -127,7 +127,11 @@ const convertScheduleToCardProps = (schedule: ScheduledEvent): ScheduleCardProps
     case "Recurring":
       return {
         variant: "recurring",
-        time: formatDate(schedule["day-time"], schedule["end-time"] ? "h:mm" : "h:mm aaa"),
+        formattedStartTime: formatDate(
+          schedule["day-time"],
+          schedule["end-time"] ? "h:mm" : "h:mm aaa",
+        ),
+        startTime: schedule["day-time"],
```

Comment:
Changed the name of `time` to `formattedStartTime` to distinguish it from `startTime` aka raw start time so that I can use the `startTime` to compare with `new Date()`